packages/editor-ui/src/shims.d.ts

```diff
@@ -13,6 +13,7 @@ declare global {
 			PROD: boolean;
 			NODE_ENV: 'development' | 'production';
 			VUE_APP_URL_BASE_API: string;
+			VUE_APP_MAX_PINNED_DATA_SIZE: string;
```

Comment:
From the looks of it, we very rarely add FE-only env vars.

This is super long running so it might've been discussed and I lost context - why did we decide to make this a FE-only env var? I'd've expected this to be in the schema, even if it only concerns the FE, but I may be wrong on this.

Reply:
It doesn't make much sense to add FE-only environment variables since they're only used at build time. After build, the variables are embedded into the bundle and never update their value again.